### PR TITLE
Added unit tests for packages/extension/src/components/dashboard/DashboardBottom.jsx

### PR DESCRIPTION
**Pull request by purvikaul**
Thank you for contributing this pull request!

Please make the PR against the `master` branch, add a description of what's changing, and link any relevant issues or PRs.

## What's changing
Added unit tests for packages/extension/src/components/dashboard/DashboardBottom.

...

## What else might be impacted? 

...

## Checklist

[ ] Unit tests (updated and/or added)
[ ] Documentation (function/class docs, comments, etc.)

Published PR with canary version: `1.0.1-canary.29.420.0`

### CODE REVIEW

File: packages/extension/test/components/dashboard/DashboardBottom.test.js [hunk 1]
```diff
@@ -20,13 +21,122 @@ const getComponent = (props = {}) => {
   )
 }
 
+
+
+const props = {
```
Comment (Celeo) [inline packages/extension/test/components/dashboard/DashboardBottom.test.js]: If this object is only going to be used in 2 tests, maybe this should either be in those 2 tests, or called something "defaultProps"?

**Comment by Celeo** [comment]
Thank you for the contribution!

File: packages/extension/test/components/dashboard/DashboardBottom.test.js (hunk 1 above)
Comment (purvikaul) [reply packages/extension/test/components/dashboard/DashboardBottom.test.js]: Done

**Review by Celeo — APPROVED** [review]
Excellent work! 🚀

**Comment by hipstersmoothie** [comment]
:rocket: PR was released in v1.0.1 :rocket:

**Pull request merged**